modules/migration-installing-legacy-operator.adoc

```diff
@@ -41,21 +41,57 @@ endif::[]
 $ sudo podman login registry.redhat.io
 ----
 
-. Download the `operator.yml` file:
+. Download the `operator.yml` file for the version of {mtc-short} you are installing:
++
+[source,terminal,subs="attributes+"]
+----
+$ sudo podman cp $(sudo podman create \
+  registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v<mtc-version-number>):/operator.yml ./
```

Comment:
The <mtc-version-number> should be an attribute, {mtc-legacy-version-z}, pointing to the current mtc version of the legacy operator.

I am not sure why you would need two steps for downloading the legacy operator.

Reply:
The PR is designed to solve the problems with the current text that is mentioned in the BZs. The current text uses the attribute you suggest so using it alone won't solve the problem. 
The solution is to present the user with the general command. The z-stream versions, both the one you suggest and the current z-stream, are given as examples, not stand-alone steps. Note that I created an attribute for the current z-stream version as well as using the existing legacy z-stream version. 
Note that no steps are being suggested, so I didn't understand your comment about the PR using two steps to download the legacy operator.

Reply:
OK, I see what you are saying. I think it would be easier for the user to follow if you structured it this way: 

```
. Download the `operator.yml` file for the version of {mtc-short} you are installing:
+
[source,terminal,subs="attributes+"]
----
$ sudo podman cp $(sudo podman create \
  registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v<mtc-version-number>):/operator.yml ./ <1>
----
<1> Specify the z-stream version number, for example, `v{mtc-legacy-version-z}`.
```
One thing that I do not understand -- why do you include the 1.7.1 operator on the procedure for 4.2 to 4.5 cluster? The older clusters only use the legacy operator.